packages/replay/src/replay.ts

```diff
@@ -839,4 +823,33 @@ export class ReplayContainer implements ReplayContainerInterface {
       saveSession(this.session);
     }
   }
+
+  /** Handler for rrweb.record.onMutation */
+  private _onMutationHandler = (mutations: unknown[]): boolean => {
+    const count = mutations.length;
+
+    const fullSnapshotOnMutationsOver = this._options._experiments.fullSnapshotOnMutationsOver || 0;
```

Comment:
Do we need to expose the implementation in the experiment name? We could shorten to `mutationLimit` -- or `fullSnapshotMutationLimit`

Reply:
makes sense, changed it!